src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -342,6 +344,16 @@ CreateCitusLocalTable(Oid relationId, bool cascadeViaForeignKeys, bool autoConve
 	 */
 	ExecuteAndLogUtilityCommandList(shellTableDDLEvents);
 
+	/*
+	 * Execute the view creation commands with the shell table. Since we want to create
+	 * views locally here, disable/enable ddl propagation. Views will be distributed via
+	 * FinalizeCitusLocalTableCreation below.
+	 */
+	tableViewCreationCommands = lcons(DISABLE_DDL_PROPAGATION, tableViewCreationCommands);
```

Comment:
at this point, the Citus local table is NOT in the metadata yet, we have not called `InsertMetadataForCitusLocalTable` / `FinalizeCitusLocalTableCreation`.

Why do we need `DISABLE_DDL_PROPAGATION` on a pg local table?